faq/index.md

```diff
@@ -53,8 +53,6 @@ A full conference ticket for two days includes access to all the amazing keynote
 
 You will also get access to all the open space discussions and other sessions during the conference. The conference ticket holders will also be eligible to join the Devsprints.
 
-The ticket also includes all the conference goodies and swags for the attendees.

```

Comment:
Thanks @anistark looks good. 
Would it be better to have a separate FAQ clearing this bit out as suggested by @inovizz on #68 
Because even though we want to say the ticket price doesn't include swag. We don't as off yet mention it anywhere specifically. I think we can add the same in this PR. If not, LGTM

Reply:
This is quick push. We don't want anymore folks to look at this and raise questions. I certainly a detailed FAQ would help once reviewed,

Reply:
I can add a separate ques for swags in #71 as I'll also have to update that to remove the swags line